src/main/webapp/profile_creation.jsp

```diff
@@ -5,7 +5,10 @@
 <t:genericpage>
     <jsp:body>
        <div class="content">
-        <form action="/profile_edit" method="POST">
+        <h3> User Registration </h3>
+        <h5> To complete your user registration please fill the fom below and click submit</h5>
+        <br>
```

Comment:
Instead of `<br>` maybe a `mb` class from Bootstrap is better for the `<h5>` before.